openfl/native/native.py

```diff
@@ -70,11 +70,15 @@ def flatten(config, return_complete=False):
 def update_plan(override_config):
     """
     Update the plan with the provided override and save it to disk.
+    If the override value is a list, keys corresponding to each list
+    value should be specified.
 
     For a list of available override options, call `fx.get_plan()`
 
     Args:
-        override_config : dict {"COMPONENT.settings.variable" : value}
+        override_config : dict {"COMPONENT.settings.variable" : value,
+                                "COMPONENT.settings.list_variable.0" : list_value_0,
```

Comment:
@mansishr does this mean that list items must be set individually? Or can a fully list be provided here? i.e.

```
"COMPONENT.settings.list_variable" : [1,2,3]
```

Reply:
List items would need to be set individually for this to work

`{"COMPONENT.settings.list_variable.0" : 1, "COMPONENT.settings.list_variable.1" : 2, "COMPONENT.settings.list_variable.2" : 3}`

Reply:
This is not ideal for long lists. I would recommend adding some logic in the update plan method to do this do this transformation programmatically:

```
final_override_config = {}
for key, val in override_config.items():
    if type(val) == list:
        for idx,v in enumerate(val):
            final_override_config[f'{key}.{idx}'] = v
    else:
        final_override_config[key] = val
```